panic while unwinding uninitialised execution pipeline

try to update fork choice while exec pipeline not ready yet:
```
INFO[05-22|15:20:04.629] [snapshots] History Stat                 blocks=0k txs=2417m txNum2blockNum="1024=14760K,1536=19814K,1544=19890K,1546=19911K,1547=0K" first_history_idx_in_db=0 last_comitment_block=0 last_comitment_tx_num=2417187500 alloc=12.8GB sys=16.5GB
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x223871b]

goroutine 369384 [running]:
github.com/ledgerwatch/erigon/eth/stagedsync.(*Sync).UnwindPoint(...)
        github.com/ledgerwatch/erigon/eth/stagedsync/sync.go:53
github.com/ledgerwatch/erigon/turbo/execution/eth1.(*EthereumExecutionModule).updateForkChoice(0xc1a8716c80, {0x31fc3e8, 0xc000ad67d0}, {0x7d, 0x98, 0x4a, 0xb8, 0xc6, 0x75, 0xc9, ...}, ...)
        github.com/ledgerwatch/erigon/turbo/execution/eth1/forkchoice.go:288 +0x2afb
created by github.com/ledgerwatch/erigon/turbo/execution/eth1.(*EthereumExecutionModule).UpdateForkChoice in goroutine 364048
        github.com/ledgerwatch/erigon/turbo/execution/eth1/forkchoice.go:96 +0x316
```

--- a/turbo/execution/eth1/forkchoice.go
+++ b/turbo/execution/eth1/forkchoice.go
@@ -284,8 +284,9 @@ func (e *EthereumExecutionModule) updateForkChoice(ctx context.Context, original
 				return
 			}
 		}
-
-		unwindToGenesis = e.executionPipeline.UnwindPoint() == 0
+		if e.executionPipeline.HasUnwindPoint() {
+			unwindToGenesis = e.executionPipeline.UnwindPoint() == 0
+		}
 
 		// Run the unwind
 		if err := e.executionPipeline.RunUnwind(e.db, wrap.TxContainer{Tx: tx}); err != nil {

